code/game/turfs/simulated/wall_attacks.dm

```diff
@@ -69,27 +68,44 @@
 
 	try_touch(user, rotting)
 
-/turf/simulated/wall/attack_generic(var/mob/user, var/damage, var/attack_message, var/wallbreaker)
+/turf/simulated/wall/attack_generic(mob/M, damage, attack_message)
+	log_and_message_admins("attacked [src]: [jumplink(src)]")
```

Comment:
```suggestion
```

Reply:
Personally I think it should be logged for when a mech breaks exterior walls or into anywhere important but that's just me 🤷